cdap-docs/_common/_themes/cdap/versions.html

```diff
@@ -0,0 +1,50 @@
+{#
+    versions.html
+    ~~~~~~~~~~~~~~~~~~~~~
+    Sphinx sidebar template: "CDAP Documentation Versions" links.
+
+    Requires a JSONP file at http://docs.cask.co/cdap/json-versions.js in the format:
+
+      versionscallback({ "development": "2.6.0-SNAPSHOT", "current": "2.5.2", "versions": ["2.5.1", "2.5.0"] });
+      
+    One development version; one current version; list of additional versions
+    the tool builder.py will generate this from scanning the current directory
+
+#}
+  <div role="note" aria-label="versions links">
+  <script type="text/javascript">
+    var this_version = "{{ release }}";
+    var writelink = function(dir, title, label){
+      style_o = '';
+      style_c = '';
+      if (dir == this_version) {
+        style_o = '<b>';
+        style_c = '</b>';
+      }
+      document.write('<li>' + style_o + '<a href="http://docs.cask.co/cdap/' + dir + '/index.html"' + 
+                     'rel="nofollow" title="' + title + '">' + label + '</a>' + style_c + '</li>');
+    }
+    var versionscallback = function(data){
```

Comment:
`versionscallback` is the only var that needs to be on the global scope. So instead of `var`, use `window.versionscallback = function() {...`